crates/aptos/src/common/types.rs

```diff
@@ -1090,6 +1125,14 @@ pub struct MovePackageDir {
     /// Select bytecode, language, compiler for Move 2
     #[clap(long)]
     pub move_2: bool,
+
+    /// Select optimization level
+    #[clap(long, value_parser = clap::value_parser!(OptimizationLevel))]
+    pub optimization_level: Option<OptimizationLevel>,
```

Comment:
Similarly, this is experimental, does it belong on all cases?  Can we add more detail on the doc comment specifying that it's experimental

Reply:
The "experiment" feature should really is just a different form of configuration, as mentioned above.  It has been convenient to select experiments, but should have been renamed long ago.  See https://github.com/aptos-labs/aptos-core/issues/12650.

Reply:
Experiment is the notion used at Google for feature flags in the orgs I worked, and I'm liking it and are happy with it.